handler/router.go

```diff
@@ -21,6 +21,10 @@ func NewHandler(services *service.Service, logger logging.Logger) *Handler {
 func (h *Handler) InitRoutes() *gin.Engine {
 	router := gin.New()
 	router.GET("/swagger/*any", ginSwagger.WrapHandler(swaggerFiles.Handler))
+	router.Use(
+		h.CorsMiddleware,
```

Comment:
Do you use userIdentity middleware?

Reply:
No, it is not an aim of this task. I plan to use this middleware later